package.json

```diff
@@ -52,25 +47,21 @@
     "@brightspace-ui/visual-diff": "^1",
     "@open-wc/testing": "^2",
     "@open-wc/testing-karma": "^4",
-    "@semantic-release/git": "^9",
+    "@webcomponents/webcomponentsjs": "^2",
     "axe-core": "^4",
     "babel-eslint": "^10",
     "chalk": "^4",
-    "d2l-license-checker": "^3.1.1",
     "deepmerge": "^4.2.2",
     "es-dev-server": "^1",
     "eslint": "^7",
     "eslint-config-brightspace": "^0.11.0",
     "eslint-plugin-html": "^6",
     "eslint-plugin-lit": "^1",
     "eslint-plugin-sort-class-members": "^1",
-    "karma-browserstack-launcher": "~1.4.0",
-    "karma-sauce-launcher": "^2",
     "lit-analyzer": "^1",
     "mocha": "^8",
     "node-sass": "^5",
     "puppeteer": "^5",
-    "semantic-release": "^17",
```

Comment:
We can remove the semantic-release config stuff below too

Reply:
Yup thanks -- removed.